test/test_mapper.py

```diff
@@ -118,6 +118,21 @@ def test_precomputed(self):
         assert graph["links"] == graph2["links"]
         assert graph["nodes"] == graph2["nodes"]
         assert graph["simplices"] == graph2["simplices"]
+        
+    def test_remove_duplicates(self, capsys):
+        mapper = KeplerMapper(verbose=1)
+        X = np.random.rand(100, 5)
+        
+        lens = mapper.project(X)
+        graph = mapper.map(
+            lens,
+            X=X,
+            cover=Cover(n_cubes=3, perc_overlap=1.5),
+            clusterer=cluster.DBSCAN(metric="euclidean", min_samples=3),
+            remove_duplicate_nodes=True   # internally asserts
```

Comment:
don't use internal asserts

Reply:
I pushed tests that don't rely on internal assets. They all pass when merged with my "fix intervals" PR